### PR TITLE
feat: [#188521509] Filter Carnival fire permit anytime action by non-essential question

### PR DESCRIPTION
**Pull request by SirSamuelJoseph**
### Ticket

This pull request resolves [#188521509](https://www.pivotaltracker.com/story/show/188521509).

### Steps to Test

1. Create a new Oscar business in the "Entertainment" industry
2. You should not see the "Get a Fire Permit for Your Carnival or Circus" anytime action
3. Go to the profile and answer yes to one of or both of the carnival non-essential questions
4. You should see the "Get a Fire Permit for Your Carnival or Circus" anytime action

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation on the engineering documentation website
- [x] I have not used any relative imports
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] I have checked for and removed instances of unused config from CMS
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/src/components/dashboard/AnytimeActionDropdown.tsx [hunk 1]
```diff
@@ -74,6 +74,11 @@ export const AnytimeActionDropdown = (props: Props): ReactElement => {
     switch (action.filename) {
       case "carnival-ride-supplemental-modification":
         return !!business?.profileData.carnivalRideOwningBusiness;
+      case "operating-carnival-fire-permit":
```
Comment (SirSamuelJoseph) [inline web/src/components/dashboard/AnytimeActionDropdown.tsx:77]: Should be added from either non-essential question

File: content/src/anytime-action-tasks/operating-carnival-fire-permit.md [hunk 2]
```diff
@@ -17,15 +15,15 @@ callToActionLink: https://www.nj.gov/dca/dfs/pdf/fire%20directory_county%20summa
...
 
-Contact the `Local Enforcing Agency (LEA)|local-enforcing-agency` in the town where your carnival or circus will be to see which fire permits you need and how to apply.
+- Setting up trailers, tents, stages, and similar structures that people will occupy
+- Setting up a `carnival or amusement ride|carnival-amusement-ride`
+- Setting up food stalls or concession stands that need a fire suppression system, such as fire extinguishers or sprinklers
+- Using an open flame or flame-producing device
+- Storing and handling certain types of flammable or combustible liquids
+- Storing and discharging fireworks
 
+Contact the `Local Enforcing Agency (LEA)|local-enforcing-agency` in the town where your carnival or circus will be to see which fire permits you need and how to apply.
```
Comment (adidner) [inline content/src/anytime-action-tasks/operating-carnival-fire-permit.md:27]: dust - a reminder that we should at some point try and unify our editor setting to align with the CMS settings at some point. (not that you should do this in this ticket or anything but a reminder for a broader disscussion)

**Review by adidner — APPROVED** [review]
Everything looks good.

**Review by somabadri — COMMENTED** [review]
lgtm!

**Pull request merged**